### PR TITLE
Add example code to the MFTF Action page

### PR DESCRIPTION
**Pull request by leviv**
## This PR is a:

- [ ] New topic
- [x] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary
In the current MFTF action documentation, many of the actions are not completely intuitive as to how they are used. These code examples and the explanations behind the examples seek to lessen this ambiguity. In additions, many developers prefer to simply copy and paste example code, and change it to suit their needs if they are unfamiliar with a particular action.

In addition, these example actions will help provide consistency for the order of action attributes, give a basis for good stepKey names, and help test writers further understand the implications behind different MFTF actions.

When this pull request is merged, it will...
Improve the ease of writing tests, and enhance the MFTF documentation

This PR adds an example section for (almost) all of the MFTF actions in the 'actions' page of the testing framework.

## Additional information

There were a few actions that are currently not used in the testing framework, and lack sufficient documentation, so I left off examples for those actions, and the pangolins team is looking at deprecating them entirely.

### CODE REVIEW

**Comment by dshevtsov** [comment]
@leviv thank you so much for this invaluable contribution!

File: guides/v2.2/magento-functional-testing-framework/release-2/test/actions.md [hunk 1]
```diff
@@ -249,6 +285,13 @@ Attribute|Type|Use|Description
 `before`|string|optional| `stepKey` of action that must be executed next.
 `after`|string|optional| `stepKey` of preceding action.
 
+#### Example:
+
+```xml
+<attachFile userInput="image.png" selector="input#imgUpload" stepKey="uploadFile"/>
+```
+This action would upload a file with the name of `image.png` to an input with the id of `imgUpload`.
```
[the remark this answers is not in the document]
Comment (leviv) [reply guides/v2.2/magento-functional-testing-framework/release-2/test/actions.md]: @tomreece has been using the attachFile action recently and said it works as expected. The preexisting documentation states that "The file must be placed in the tests/_data directory.", but I can update the example description if you think that is not clear enough.

[the remark this answers is not in the document]
Comment (leviv) [reply guides/v2.2/magento-functional-testing-framework/release-2/test/actions.md]: went ahead and updated the example explanation

File: guides/v2.2/magento-functional-testing-framework/release-2/test/actions.md [hunk 2]
```diff
@@ -356,16 +475,16 @@ Attribute|Type|Use|Description
 `before`|string|optional| `stepKey` of action that must be executed next.
 `after`|string|optional| `stepKey` of preceding action.
 
+```xml
+<amOnPage url="/login" stepKey="goToLoginPage"/>
+<comment userInput="I am on the login page" stepKey="loginPageComment"/>
+```
+This action will leave a comment of `I am on the login page` after the `amOnPage` action in the generated PHP file.
```
[the remark this answers is not in the document]
Comment (leviv) [reply guides/v2.2/magento-functional-testing-framework/release-2/test/actions.md]: Fixed

File: guides/v2.2/magento-functional-testing-framework/release-2/test/actions.md [hunk 3]
```diff
@@ -845,6 +1214,13 @@ Attribute|Type|Use|Description
 `before`|string|optional| `stepKey` of action that must be executed next.
 `after`|string|optional| `stepKey` of preceding action.
 
+#### Example:
+
+```xml
+<loadSessionSnapshot userInput="savedSnapshot" stepKey="loadSnapshot"/>
```
[the remark this answers is not in the document]
Comment (leviv) [reply guides/v2.2/magento-functional-testing-framework/release-2/test/actions.md]: Tested and it works as expected

File: guides/v2.2/magento-functional-testing-framework/release-2/test/actions.md [hunk 4]
```diff
@@ -1038,6 +1518,13 @@ Attribute|Type|Use|Description
 `before`|string|optional| `stepKey` of action that must be executed next.
 `after`|string|optional| `stepKey` of preceding action.
 
+#### Example:
+
+```xml
+<saveSessionSnapshot userInput="savedSnapshot" stepKey="saveCurrentCookies"/>
```
[the remark this answers is not in the document]
Comment (leviv) [reply guides/v2.2/magento-functional-testing-framework/release-2/test/actions.md]: Tested and it works as expected

**Comment by leviv** [comment]
@KevinBKozan feedback addressed, let me know if there's any other concerns

**Pull request merged**